Backpack/src/main/java/net/skyscanner/backpack/calendar/presenter/BpkCalendarController.kt

```diff
@@ -60,6 +60,10 @@ abstract class BpkCalendarController {
   // Shall we use DateFormat.getDateInstance instead?
   internal fun getLocalizedDate(date: Date, pattern: String): String = SimpleDateFormat(pattern, locale).format(date)
 
+  open fun isToday(year: Int, month: Int, day: Int): Boolean {
```

Comment:
Why does this needs to be all the way up in the controller and specially be left `open`? Do we expect this behavior to be defined by the consumer of this library? Sounds more like a helper method to me 🤔

Reply:
I need this for the snapshot testing. We need "today" to be always the same day during snapshots. Couldn't find a better way of doing it.